monkestation/code/datums/ai/idle_behaviors/idle_dukeman.dm

```diff
@@ -0,0 +1,19 @@
+/datum/idle_behavior/idle_dukeman/dukeman
```

Comment:
alright I'll get to fixing that real quick rn

Reply:
mainly set the path type to "idle_dukeman/dukeman" instead of "idle_monkey/dukeman" cause I was extra precocious of accidentally causing the bug again originally but yeah